web-ui/src/main/resources/catalog/components/viewer/wfs/WfsDirective.js

```diff
@@ -123,14 +123,17 @@
         templateUrl: '../../catalog/components/' +
             'viewer/wfs/partials/download.html',
         link: function(scope, element, attrs, ctrls) {
-          scope.capabilities;
+          scope.capabilities,scope.exceptionDetail,scope.exception;
```

Comment:
Is this required? the values are initialised in `init()`, I don't think this line is required.

Reply:
+1